migrations/versions/6fc1a99cefd5_create_coastlines.py

```diff
@@ -0,0 +1,63 @@
+"""Create coastlines
+
+Revision ID: 6fc1a99cefd5
+Revises: d632ea3490c4
+Create Date: 2022-05-21 14:48:54.624485
+
+"""
+from alembic import op
+from geoalchemy2.types import Geography
```

Comment:
This [blog post](https://medium.com/coord/postgis-performance-showdown-geometry-vs-geography-ec99967da4f0) drove my decision to set it as Geography for the moment. We can always optimize it in the future.

Reply:
Actually that's pretty cool to know. It was a good read and it makes sense, penalty is very negligible especially since we're probably going to return at most like 300 rows. Totally agree with this move.